.vitepress/config.js

```diff
@@ -55,7 +55,7 @@ export default defineConfig({
 				link: 'https://www.dovecot.org/'
 			},
 			{
-				text: 'Repositories',
+				text: 'Download repositories',
```

Comment:
Just that Repositories can also mean Code Repositories (like, github).

Reply:
Fair point, but still don't like the wordiness.

How about "Packages" since that's explaining what we are doing with the repos (building packages for specific OS's and distributing them via package managers)?

Reply:
i can add commit for packages.